### PR TITLE
Fixed get locale in dynamic controller

### PR DESCRIPTION
**Pull request by dominikmatt**

### CODE REVIEW

**Comment by dominikmatt** [comment]
@alexander-schranz

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## dev-develop
 
+ - BUGFIX      #88    Fixed get locale in dynamic controller.
```
Comment (alexander-schranz) [inline CHANGELOG.md]: remove `.`

add new changelog release

```
## 0.2.1
 
  - BUGFIX      #88    Fixed get locale in dynamic controller

## 0.2.0

     ...
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 ## dev-develop
```
Comment (alexander-schranz) [inline CHANGELOG.md]: remove

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
...
 
+  - BUGFIX     #88    Fixed get locale in dynamic controller
```
Comment (alexander-schranz) [inline CHANGELOG.md]: remove

**Pull request merged**